executor/api/util/utils.go

```diff
@@ -141,3 +141,26 @@ func GetEnvAsBool(key string, fallback bool) bool {
 
 	return fallback
 }
+
+type SslKakfa struct {
+	KafkaSslClientCertFile string
+	KafkaSslClientKeyFile  string
+	KafkaSslCACertFile     string
+	KafkaSslClientKeyPass  string
+}
+
+func (o SslKakfa) String() string {
+	return "SslKakfa"
+}
+
+var KafkaSecurityProtocol = GetEnv("KAFKA_SECURITY_PROTOCOL", "")
```

Comment:
:thought_balloon: Probably worth having this as a function, so it calls `GetEnv` every time rather than whenever the module is instantiated.  That'd be useful for testing.

Reply:
Ah yeah it makes sense 😅, I removed the var and only use a function now.